src/context.ts

```diff
@@ -12,3 +12,10 @@ export const createContext = (ctx: any) => {
     set: (ctx: any) => (_ctx = ctx)
   }
 }
+
+export const useContext = (ctx: any) => {
+  let _ctx = ctx
+  if (_ctx && typeof(_ctx.get) == "function") {
```

Comment:
- I prefer Strict Equality Comparison (===)
- `"function"` should be in single quotes

Reply:
Done.